distribution/ddf-common/src/main/resources-filtered/etc/application-definitions/solr.json

```diff
@@ -2,6 +2,7 @@
   "name": "catalog-solr-app",
   "description": "The Solr Catalog Provider (SCP) is an implementation of the CatalogProvider interface using Apache Solr ${solr.version} as a data store.\nThe SCP supports extensible metacards, fast/simple contextual searching, indexes xml attributes/CDATA sections/XML text elements, contains full XPath support, works with an embedded local Solr Server, and also works with a standalone Solr Server.::Solr Catalog",
   "bundleLocations": [
+    "mvn:org.codice.thirdparty/jts/${jts.bundle.version}",
```

Comment:
why is this added to the application definition?

Reply:
to be consistent with how the feature is composed of https://github.com/lamhuy/ddf/blob/master/catalog/solr/catalog-solr-app/src/main/resources/features.xml#L43